addons/sale_product_configurator/static/src/js/product_configurator_dialog/product_configurator_dialog.js

```diff
@@ -48,7 +49,7 @@ export class ProductConfiguratorDialog extends Component {
             currencyId: this.props.currencyId,
             addProduct: this._addProduct.bind(this),
             removeProduct: this._removeProduct.bind(this),
-            setQuantity: this._setQuantity.bind(this),
+            setQuantity: useDebounced(this._setQuantity.bind(this), 1000, true),
```

Comment:
That's not really what I had in my mind. I meant for you to create a new method that would be the one denounced. I don't think debouncing `setQuantity` in every case is good.

Also, I'd use less time like 500ms max. 
```suggestion
            setQuantity: useDebounced(this._setQuantity.bind(this), 300),
```